config/src/main/scala/bloop/config/Config.scala

```diff
@@ -213,6 +213,8 @@ object Config {
       modules: List[Module]
   )
 
+  case class Settings(semanticDBVersion: String)
```

Comment:
We might actually want to leave the `semanticdbVersion` field here but make it an `Option`. This could be a escape hatch for users that know what they are doing and want to force a concrete version, otherwise we pick `latest.release` if `semanticdbVersion` is missing.